src/Localization/Localization/src/ResourceManagerStringLocalizer.cs

```diff
@@ -54,7 +54,9 @@ public class ResourceManagerStringLocalizer : IStringLocalizer
         /// </summary>
         public ResourceManagerStringLocalizer(
             ResourceManager resourceManager,
+#pragma warning disable PUB0001 // Pubternal type IActivator in public API
```

Comment:
@mkArtakMSFT who should I talk to regarding these pubternal APIs in Localization?

Reply:
Is the idea here that these constructors and their PubTernal types should become internal and the tests should use IVT? That seems reasonable, file an issue and we can go from there.

The pragma comments seem to be referring to an `IActivator` type which is not actually used here though.

Reply:
The comments are my bad, copy-pasta. The question is how should we deal with the pubternal types. Should we make the types/methods public? Or should we make them internal? We did a pass in aspnetcore to remove pubternals but it seems like we didn't do so for extensions. We just need to go through the same process again for Localization.

Reply:
Suggest filing an issue describing the proposed changes in detail e.g. "burn pub-ternal with fire and make them all `internal`" or whatever makes sense. Then, mark for API review.

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/19081